src/methods/status.rs

```diff
@@ -1,3 +1,27 @@
+//! Requests the status of the connected RPC node.
+//!
+//! Returns the status of the connected RPC node (sync status, nearcore node version, protocol version, etc), and the current set of validators.
```

Comment:
Let's deduplicate this.

```suggestion
//! Requests the status of the connected RPC node.
//!
//! This includes information about sync status, nearcore node version, protocol version, the current set of validators, etc.
```